commit/merkleroot/rmn/crypto_test.go

```diff
@@ -0,0 +1,133 @@
+package rmn
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/commit/merkleroot/rmn/rmnpb"
+)
+
+var (
+	empty32ByteArr [32]byte
+	empty20ByteArr [20]byte
+)
+
+func Test_verifyObservationSignature(t *testing.T) {
+	offchainPublicKey := "0x137d557b609823f3f8f841265d5797e0fe97892b496764af326fd62a82d881aa"
+
+	expSig := "edc308a23ce4aabf7518ad38778447e407b89b13011f7abc10e7641a51728e9ec6dd49a014ca238d8ba5d91fe9aff115d" +
```

Comment:
Is this a shared test case w/ RMN? If so lets add a comment indicating that.

Reply:
added a comment